lisa/util/__init__.py

```diff
@@ -109,6 +109,19 @@ def __str__(self) -> str:
         return message
 
 
+class UnsupportedArchException(LisaException):
```

Comment:
use full name to make it clearer. `UnsupportedArchitectureException`

Reply:
Done